### ISSUE TITLE
glibc version wrong in container

### ISSUE THREAD

**Issue by rothgar** [w4/bin#60]
The example container when built and run gives the following error
```
/pastebin: /lib/x86_64-linux-gnu/libc.so.6: version `GLIBC_2.33' not found (required by /pastebin)                                                                                            
/pastebin: /lib/x86_64-linux-gnu/libc.so.6: version `GLIBC_2.32' not found (required by /pastebin)                                                                                            
/pastebin: /lib/x86_64-linux-gnu/libc.so.6: version `GLIBC_2.34' not found (required by /pastebin)
```
The version of the libc6 package available in debian bullseye is `2.31-13+deb11u7` which is too old to meet these requirements.

I also checked in the new debian bookworm release but the libc6 package there is too new `2.36-9+deb12u3`

Is there a hard requirement on those versions or a way to install the required version without compiling from source?

**Comment by rothgar** [comment]
I also tried compiling glibc 2.34 inside the container with this
```
apt-get install gawk bison gcc make wget tar python3 -y
wget -c https://ftp.gnu.org/gnu/glibc/glibc-2.34.tar.gz
tar -zxvf glibc-2.34.tar.gz && cd glibc-2.34
mkdir glibc-build && cd glibc-build
../configure --prefix=/lib/x86_64-linux-gnu
make
make install
```
But I get the same error.

**Comment by w4** [comment]
This should be fixed by #61, let me know how you get on

**Comment by rothgar** [comment]
Sorry for not following up. Yes, this worked for me.
